openchallenges/data_integration_props.py

```diff
@@ -0,0 +1,13 @@
+from aws_cdk.aws_scheduler_alpha import ScheduleExpression
+
+
+class DataIntegrationProps:
```

Comment:
we've generally been adding property classes to `ServiceProps.py` file in this repo and in the other OC like AWS CDK repos.  You can also annotate the class as a [dataclass](https://docs.python.org/3/library/dataclasses.html) which will handle the init for you.

Reply:
I will do that.

Reply:
Should `@dataclass` be applied to `ServiceProps` too?

Reply:
Done